### DISCUSSION TITLE
code for free - Onde achar projetos para ajudar

### DISCUSSION THREAD

**Discussion by letanure** [frontendbr/forum#662] [Discussão]
Pessoal, estou querendo fazer alguns projetos diferentes, aplicar algumas coisas novas, gastar algum tempo ou mesmo só ensinar, entäo estou procurando algum lugar que tenham projetos que eu possa fazer ou ajudar, de graça.
Nåo exatamente libs, ferramentas open source, mas projetos mais reais.

Alguém sabe onde posso achar algo assim? de preferencia com boa definicao de requisitos, mais do que "faz um site aí!"

**Comment by geovanerocha** [comment]
Uma coisa legal, mas não é código, é a tradução do frontendbook :) ia ajudar uma boa galera que tá começando e tem dificuldades no ingles

**Comment by calaca** [comment]
@geovanerocha queria ajudar nesse projeto também. Você tem o link dele, por favor?

**Comment by letanure** [comment]
eu estava querendo mais projetos reais, fazer um sistema, um hotsite, nao so traducao.

algo pra praticar coisas diferentes do que faco no dia a dia, desde o basico até usar coisas mais complexas

**Comment by kivervinicius** [comment]
@letanure se sente a vontade em ensinar/"mentorar"? Seja um mentor do nosso projeto @training-center

**Comment by geovanerocha** [comment]
@calaca https://github.com/kvnol/front-end-handbook-2017/tree/pt-br/translations/pt_BR

**Comment by heliomsolivas** [comment]
@letanure se quiser ajudar, to reunindo um povo, mais pra aprendizado mesmo
https://github.com/heliomsolivas/vue-estudos

**Comment by letanure** [comment]
@kivervinicius vou dar uma olhada la com calma, valeu! sempre prometo ajudar mais como antigamente, mas nao dura muito o ímpeto.

mas o que estou procurando mesmo é fazer um trabalho de graca pra aguem, só pela diversao mesmo. sou dev faz 15 anos, front desde 2006, hj trabalho como tech leader front end numa empresa em Berlin e quero fazer algo diferente. freela é um saco procurar um bem especificado, entao doar trabalho pra alguem que precise, essa é a ideia.

ajudar em projetos open source, faco de vez em quando, mas agora queria alguns desafios mais praticos e menos teóricos, entendem?

**Reply by Erlingsjunior** [reply]
Eu tenho um projeto para você olhar !

**Comment by CinthiaQueiro** [comment]
Esses dias eu também iria fazer a mesma pergunta. Se existia um repositório mais centralizado tipo o frontendbr onde são incluídas issues com projetos open para contribuição que vocês utilizam para "caçar" jobs para contribuir. Confesso que sempre fui extremamente tímida para fazer contribuições e apesar de um tempo na área nunca tive "coragem" de me envolver, mas estava querendo começar a fazer contribuições também.

**Comment by kvnol** [comment]
Galera, aqui mesmo no fórum Front-end Brasil temos um repositório com vários projetos open source precisando de ajuda: https://github.com/frontendbr/open-source/issues :heart:

**Comment by felipefialho** [comment]
Fala @letanure, como o @kvnol disse temos uma iniciativa nesse sentido aqui mesmo.

Acho que vale dar uma divulgada inclusive.

https://github.com/frontendbr/open-source/issues

**Comment by CinthiaQueiro** [comment]
Legal! Vou dar uma olhada. :)

Em 15 de ago de 2017 11:21, "Felipe Fialho" <notifications@github.com>
escreveu:

**Comment by calaca** [comment]
@geovanerocha ah, eu achei que fosse outro livro. Já estou ajudando nesse. Obrigada mesmo assim. :)

**Comment by letanure** [comment]
valeu @LFeh 
ver se tem esses projetos pontuais por lá

**Comment by rubenmarcus** [comment]
@letanure  cara, uma forma seria ser mentor, ou até sei lá procurar uma ong ai na Europa ou aqui no Brasil que precise.  Eu sou voluntário como fotografo num ong aqui de SP e fiz o site deles também achei pelo https://www.atados.com.br/

https://www.atados.com.br/explore/vagas/  Ai vi que tem algumas pra devs... desde site a sistemas , talvez seja isso que você queira


Aqui umas legais
https://www.atados.com.br/vaga/desenvolvedor-front-end
https://www.atados.com.br/vaga/zica-da-programacao
https://www.atados.com.br/vaga/programador-anjo


Eu tenho muito essa mentalidade também de repassar conhecimento,e em umas conversas que tive com a organizadora da ONG que trabalho, ela falou uma verdade que ficou na minha cabeça, comentavamos de algumas oportunidades de trabalho que eu tinha pra trabalhar fora do Brasil e ela disse o seguinte:
-Lá na Europa ou EUA, eles já tem gente o suficiente, não precisam de você, o pessoal da sua área, tem cursos e fóruns e meios de se organizar e se desenvolver, o pessoal daqui da periferia, a mulecada que não tem acesso, e nem conhecimento, que precisa de gente igual você, pra ter uma mudança de perspectiva.

E eu concordo com ela. Acho muito valido pessoal do open source, e cursos e tudo mais, isso incentiva ainda mais criação de coisas legais, mas também é muito bom levar o conhecimento que temos a quem nem se quer tem idéia de que existe.

E isso pode ser feito de diversas formas, seja um site ou sistema para divulgar alguem que já faz esse trabalho ou mesmo presencialmente.

**Comment by benguimaraes** [comment]
@rubenmarcus muito bacana o site que você sugeriu.

**Comment by Zutzsa** [comment]
@letanure 
Existe um site chamado "Freecodecamp" que é voltado a ensinar Front-end e Back-end para novatos, eles criaram alguns projetos chamados de "Opensource for good" que visa possibilitar ganhar experiencia real e ao mesmo tempo ajudar umas organizações sem fins lucrativos.

Segue abaixo o link feito no site deles falando sobre o projeto e a lista de repositórios nesse projeto:

[Open source for good](https://medium.freecodecamp.org/open-source-for-good-now-its-easier-than-ever-to-code-for-a-cause-91901096c4e0)

[Lista de repositórios](https://www.freecodecamp.org/nonprofits/)


@CinthiaQueiro 

O site [Openhatch](http://openhatch.org/search/) tem uma lista com vários projetos legais

**Comment by phhoffmann** [comment]
Galera, no [Adote um briefing](http://adoteumbriefing.org/) tem várias coisas, não só pra desenvolvedores, tbm pra designers, etc.

**Comment by guilhermepontes** [comment]
https://www.codetriage.com/

**Comment by camilatigre** [comment]
https://github.com/camilatigre/listamaravilhosaopensource 

Não sei se ajuda, mas se encontraram esses projetos reais pra colaborar seria legal colocar na lista ((:

**Comment by AhCamargo** [comment]
Boa tarde, Sou novo em Front-end, quero participar em algum projeto real, para que eu possa ajudar e aprender.  alguém indica um repo?

**Comment by herberthenrique** [comment]
@acamargo2017 Da uma olhada nesse repositório, tem bastante desafios de front para você ir treinando :)
https://github.com/LFeh/frontend-challenges

**Comment by alinebastos** [comment]
Tem bastante coisas aqui também:
https://github.com/alinebastos/frontend-practice
:)

**Comment by AhCamargo** [comment]
@herberthenrique  e @alinebastos  Obrigado.

**Comment by felipefialho** [comment]
@acamargo2017 [Aproximando seus estudos de cenários reais](https://www.felipefialho.com/blog/2015/aproximando-seus-estudos-de-cenarios-reais)

**Comment by letanure** [comment]
valeu pessoal. vou olhar de novo todos os links,  
nao estou procurando para aprender (apesar de ser bom, se possivel...) e sim para fazer algo diferente mesmo
estou procurando algum projeto mais complexo para participar ou criar algo para alguma ONG que precise

**Comment by JoseCage** [comment]
@letanure Se tiver interesse pode dar uma olhada no projeto @doesangueorg, em particular [doesangueorg/doesangue-webapp](https://github.com/doesangueorg/doesangue-webapp)

**Comment by droliv** [comment]
Boa noite galera, eu fundei um projeto chamado Code Solidário que é para desenvolver sites e sistemas gratuitamente para ONGs, então se estiverem interessados em colaborar, até o momento eu sou meio que iniciante, nosso site precisa de melhorias e não temos nenhuma instituição sendo atendida ainda, tudo muito no começo, mas com muita vontade pra fazer acontecer.
Vem com a gente!
https://github.com/CodeSolidario/CodeSolidario.github.io

**Comment by diegosurita** [comment]
Legal @thedannyreis, eu estava pensando em fazer a mesma coisa. Vou dar uma olhada no repositório e ver no que eu posso contribuir.

**Comment by Fabio-Pina** [comment]
> valeu pessoal. vou olhar de novo todos os links,
> nao estou procurando para aprender (apesar de ser bom, se possivel...) e sim para fazer algo diferente mesmo
> estou procurando algum projeto mais complexo para participar ou criar algo para alguma ONG que precise



> Pessoal, estou querendo fazer alguns projetos diferentes, aplicar algumas coisas novas, gastar algum tempo ou mesmo só ensinar, entäo estou procurando algum lugar que tenham projetos que eu possa fazer ou ajudar, de graça.
> Nåo exatamente libs, ferramentas open source, mas projetos mais reais.
> 
> Alguém sabe onde posso achar algo assim? de preferencia com boa definicao de requisitos, mais do que "faz um site aí!"

Olá amigo boa noite. Eu segui você tenho algumas perguntas para fazer. Tem algum contato? Aguardo respostas. Obrigado.

**Comment by Fabio-Pina** [comment]
> @letanure cara, uma forma seria ser mentor, ou até sei lá procurar uma ong ai na Europa ou aqui no Brasil que precise. Eu sou voluntário como fotografo num ong aqui de SP e fiz o site deles também achei pelo https://www.atados.com.br/
> 
> https://www.atados.com.br/explore/vagas/ Ai vi que tem algumas pra devs... desde site a sistemas , talvez seja isso que você queira
> 
> Aqui umas legais
> https://www.atados.com.br/vaga/desenvolvedor-front-end
> https://www.atados.com.br/vaga/zica-da-programacao
> https://www.atados.com.br/vaga/programador-anjo
> 
> Eu tenho muito essa mentalidade também de repassar conhecimento,e em umas conversas que tive com a organizadora da ONG que trabalho, ela falou uma verdade que ficou na minha cabeça, comentavamos de algumas oportunidades de trabalho que eu tinha pra trabalhar fora do Brasil e ela disse o seguinte:
> -Lá na Europa ou EUA, eles já tem gente o suficiente, não precisam de você, o pessoal da sua área, tem cursos e fóruns e meios de se organizar e se desenvolver, o pessoal daqui da periferia, a mulecada que não tem acesso, e nem conhecimento, que precisa de gente igual você, pra ter uma mudança de perspectiva.
> 
> E eu concordo com ela. Acho muito valido pessoal do open source, e cursos e tudo mais, isso incentiva ainda mais criação de coisas legais, mas também é muito bom levar o conhecimento que temos a quem nem se quer tem idéia de que existe.
> 
> E isso pode ser feito de diversas formas, seja um site ou sistema para divulgar alguem que já faz esse trabalho ou mesmo presencialmente.

Olá, tudo bem? Eu lhe enviei um e-mail se puder me responder agradeço, obrigado.

**Comment by carioca808080** [comment]
Olá tudo bem, meu nome é Ismael tenho 16 anos e gosto de ajudar na programação e em informática, com preços bem encontar 😊 ( sexta/domingo )

**Comment by leorsoccol** [comment]
Boa galera Galera.  Desenvolvi um app através de uma inteligência artificial.   Mas queria muito criar um APP para postar na play store gratuitamente na minha conta.  Trata-se de um app para a pessoa verificar os sintomas da depressão.  Quem estiver disposto a ajudar.  Obrigado
